src/components/layout/AddToWishlistButton/AddToWishlistButton.js

```diff
@@ -80,7 +80,7 @@ const AddToWishlistButton = ({ album, wishListId }) => {
     <>
       <Tooltip title={isAdded ? 'Remove from wishlist' : 'Add to wishlist'}>
         <IconButton
-          color="secondary"
+          color="error"
```

Comment:
This is fine; but a bit misleading to call this `error`. And if we ever want to change the color of `"error"` in our theme for the app, then it might be a bit of a surprise that it also changes the wishlist icon color. We can avoid this by creating a new color variable for `wishlist` in our MUI light and dark themes in `App.js`:

```javascript
const lightTheme = createTheme({
    palette: {
      mode: 'light',
      primary: { main: '#000000' },
      background: { default: '#C0C0C2', paper: '#ECECEC' },
      // etc any other customizations that we want to make for light mode
      wishlist: {
        main: '#ff4536',
      },
    },
    typography: {
      fontFamily: 'Poppins, sans-serif',
    },
  });
```

and then here we can refer to it with:

```suggestion
          color="wishlist"
```

Reply:
thank you! done!